sys/include/net/ieee802154.h

```diff
@@ -78,12 +78,24 @@ extern "C" {
 /** @} */
 
 /**
- * @brief   Flag for @ref ieee802154_set_frame_hdr to indicate to ignore @p dst
- *          and @p dst_len and send broadcast.
- * @note    This flag is RIOT internal and shall not be used in the FCF of
- *          packets send over the air
+ * @brief   Special address defintions
+ * @{
+ */
+/**
+ * @brief   Static initializer for broadcast address
+ */
+#define IEEE802154_ADDR_BCAST               { 0xff, 0xff }
```

Comment:
mhm, this is a matter of taste --and grouping things.

I was thinking `IEEE802154_BCAST_ADDR` and `IEEE802154_BCAST_ADDR_LEN` together, so its an broadcast address which has _address length_ (not _broadcast length_).

But, what matters most is that we get this PR/ fix through, and I don't want to start a naming discussion - such issues are all over the place, if we would look deep enough 😬